polkadot/node/core/pvf/src/execute/queue.rs

```diff
@@ -305,8 +305,6 @@ impl Queue {
 		for hash in &update.deactivated {
 			let _ = self.active_leaves.remove(&hash);
 		}
-
-		gum::debug!(target: LOG_TARGET, size = ?self.active_leaves.len(), "Active leaves pruned");
```

Comment:
We could also make this a trace to get less noise.

Reply:
But if you are 100% sure we never need to debug this in prod, I guess it's fine

Reply:
Yep, we could downgrade this or move to a separate log-target for "extra" details, maybe it will come in handy in the future, up to you 🙏